ros_network_viz/ros_network_viz.py

```diff
@@ -111,6 +127,21 @@ def update_path(self, source_point, target_point):
 
         self.setPath(path)
 
+    # PyQt method override
+    def paint(self, painter, option, widget):
+        super().paint(painter, option, widget)
```

Comment:
Minor point, but the way I've usually seen this done for the overridden methods is to first do the custom thing you want to do, and then call the `super().method()`.  You'll notice that in all of the other overridden methods in this file, that's the way it is done.  Did you have a particular reason to do it the other way here?

Reply:
Haven't reviewed the PR, just passing through.

I'm used to seeing `super()` used first in the method, but that's probably because I usually only see it in `__init__`. There initializing the base classes first before the new class makes sure everything from them is ready to be used.

Reply:
I think it would depend on the method. For __init__ it makes sense that the call to super is first as one wants to contruct the base class before the dervied class. In the case of drawing, I think it also make sense to draw/paint the base class first before adding to it. In this case, I'm adding an arrow to the endpoint of the line. For example, imagine the arrowhead was black and the line was another color. If I was to draw the arrow first, the line might then be drawn on top of the arrowhead (unless, of course it was only drawn to the beginning of the arrow). Anyways, I would tend to default to calling super on the base class in most cases, unless there was a reason to do it otherwise.

Reply:
That's fair enough.  I guess it is a case-by-case basis.